docs/source/Usage.md

```diff
@@ -309,11 +309,29 @@ Pivots are designed to create a job from another job. This plugin allows the use
 
 This is a "SOAR" feature that allows the users to connect multiple analysis together.
 
-Right now the support for this kind of plugin in the GUI is very limited, while the backend is fully operative. We are working on the frontend.
-
 #### List of pre-built Pivots
 None
 
+You can build your own custom Pivot with your custom logic with just few lines of code. See the [Contribute](https://intelowl.readthedocs.io/en/latest/Contribute.html#how-to-add-a-new-pivot) section for more info.
+
+#### Creating Pivots from the GUI
+
+From the GUI, the users can pivot in two ways:
+- If a Job executed a [Visualizer](#visualizers), it is possible to select a field extracted and analyze its value by clicking the "Pivot" button (see following image). In this way, the user is able to "jump" from one indicator to another.
+![img.png](../static/pivot_job_report.png)
+
+- Starting from an already existing [Investigation](#investigations-framework), it is possible to select a Job block and click the "Pivot" button to analyze the same observable again, usually choosing another [Playbook](#playbooks) (see following image)
+![img.png](../static/pivot_investigation_report.png)
+
+In both cases, the user is redirected to the Scan Page that is precompiled with the observable selected. Then the user would be able to select the [Playbook](#playbooks) to execute in the new job.
+After the new Job is started, a new [Investigation](#investigations-framework) will be created (if it does not already exist) and both the jobs will be added to the same Investigation.
```

Comment:
please add a simple image for this too, maybe using one the ip addresses you use after

Reply:
@carellamartina I wait for this, the i'll merge the pr